examples/pod_runtime.json

```diff
@@ -46,8 +49,14 @@
                 ]
             },
             "mounts": [
-                {"volume": "database", "path": "/mnt/db"},
-                {"volume": "buildoutput", "path": "/mnt/out"}
+                {
+                    "volume": "dbvolume",
```

Comment:
changed this name to be more explicit about the difference between mountpoints/mounts/volumes